submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/SubmarineConfiguration.java

```diff
@@ -480,8 +481,8 @@ public boolean getBoolean(String envName, String propertyName, boolean defaultVa
   public void updateConfiguration(String name, String value) {
     properties.put(name, value);
   }
-
-  public enum ConfVars {
+/*
+  public enum SubmarineConfVars {
```

Comment:
Can you delete `public enum SubmarineConfVars {....}` code block?

Reply:
Thank @liuxunorg for your review. I have fixed the issue.